package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name"            : "eslint-config-defaults",
```

Comment:
**Note**: `npm --save` formats `package.json`, so we should just let it... (keeping the spaces removed)